api/controllers/connection.js

```diff
@@ -0,0 +1,174 @@
+const Sequelize = require('sequelize');
+const models = require('../../database/models');
+const response = require('../helpers/response');
+
+module.exports = {
+  async getConnectionsRequestsForUser(req, res) {
+    const { userId } = req.params;
+    try {
+      const connections = await models.Connections.findAll({
+        attributes: [
+          'id',
+          'sender_user_id',
+          'request_user_id',
+          'accepted',
+          'pending'
+        ],
+        include: [
+          {
+            model: models.Users,
+            attributes: [
+              'id',
+              'first_name',
+              'last_name',
+              'username',
+              'profile_picture'
+            ],
+            as: 'request_user'
+          },
+          {
+            model: models.Users,
+            attributes: [
+              'id',
+              'first_name',
+              'last_name',
+              'username',
+              'profile_picture'
+            ],
+            as: 'sender_user'
+          }
+        ],
+        order: [['updated_at', 'DESC']],
+        where: {
+          [Sequelize.Op.or]: [
+            { sender_user_id: userId },
+            { request_user_id: userId }
+          ],
+          rejected: false,
+          accepted: false
+        }
+      });
+      const sent = connections
+        .filter(c => c.sender_user_id === parseInt(userId, 10))
+        .map(m => {
+          return {
+            id: m.id,
+            accepted: m.accepted,
+            pending: m.pending,
+            connection: m.request_user
+          };
+        });
+      const recieved = connections
```

Comment:
Typo `received`

Reply:
I will fix that.... But it has to be mergers now so Katy can work with it on the front end.... I'm away from my PC. Also don't delete this branch after merging